app/pages/collections/list.cjsx

```diff
@@ -50,7 +50,7 @@ List = React.createClass
   listCollections: (props) ->
     query = {}
     if props.params?.collection_owner?
-      query.owner = props.params.collection_owner
+      query.current_user_roles = "owner,contributor,collaborator"
```

Comment:
This is still a bit wonky. Logged in as my staging user, srallen086, and viewing `/collections/zwolfenbarger` I see my own collections. `zwolfenbarger` in the route is the `prop.params.collection_owner`. I think the conditional here will need to be reworked since the collection owner won't always be the logged in user.

<img width="1212" alt="screen shot 2017-02-16 at 11 09 25 am" src="https://cloud.githubusercontent.com/assets/5016731/23032253/6d215556-f439-11e6-8268-58a86cf725d2.png">

The `query.owner = props.params.collection_owner` will need to remain. There needs to be a way to be able to fetch the roles for the `:collection_owner` user param.